kate/src/lib.rs

```diff
@@ -10,23 +12,25 @@ pub const LOG_TARGET: &str = "kate";
 pub type Seed = [u8; 32];
 
 pub mod config {
+	use super::{BlockLenghtColumns, BlockLenghtRows};
+
 	pub const SCALAR_SIZE_WIDE: usize = 64;
 	pub const SCALAR_SIZE: usize = 32;
 	pub const DATA_CHUNK_SIZE: usize = 31; // Actual chunk size is 32 after 0 padding is done
-	pub const EXTENSION_FACTOR: usize = 2;
-	pub const PROVER_KEY_SIZE: usize = 48;
+	pub const EXTENSION_FACTOR: BlockLenghtRows = BlockLenghtRows(2);
+	pub const PROVER_KEY_SIZE: BlockLenghtRows = BlockLenghtRows(48);
```

Comment:
Same here, this is number of bytes, not sure it has same constraints as number of rows (if I don't miss something).